src/live/logger/index.js

```diff
@@ -0,0 +1,118 @@
+const origStrOutWrite = process.stdout.write;
+
+export default class Logger {
+    constructor () {
+        this.testingStarted = false;
+        this.aborted        = false;
+        this.running        = false;
+        this.watching       = true;
+
+        process.stdout.write = (...args) => this._onStdoutWrite(...args);
```

Comment:
If I understand correctly, we have to use monkeypatching here only to catch the moment when a `TestRun` is aborted https://github.com/DevExpress/testcafe/pull/3222/files#diff-5c711b0cfadee9cf2117801b2ead2e71R7

Maybe we can emit some event instead of this hack?